### ISSUE TITLE
A11Y : Footer Contact Info 

### ISSUE THREAD

**Issue by jcsparks** [CuBoulder/tiamat-theme#911]
Looks like the content in the footer is currently wrapped in an `<h5>` tag.  This was likely done for stying reasons.  It should be semantically an `<h2>` and styled appropriately to look visually as it does with CSS.  

It's currently in a `<h2>` in D7.
